linkml/generators/fhirgen.py

```diff
@@ -0,0 +1,756 @@
+"""fhirgen: FHIR-related generators
```

Comment:
Name of file: I called it `fhirgen.py`, but I only intend to do Enum->ValueSet. I can rename it to something more specific if desired.